modules/rostests/rosautotest/CWineTest.cpp

```diff
@@ -64,11 +64,27 @@ CWineTest::GetNextFile()
     WIN32_FIND_DATAW fd;
 
     /* Did we already begin searching for files? */
-    if(m_hFind)
+    if (m_hFind)
     {
         /* Then get the next file (if any) */
-        if(FindNextFileW(m_hFind, &fd))
-            FoundFile = true;
+        if (FindNextFileW(m_hFind, &fd))
+        {
+            // printf("cFileName is '%S'.\n", fd.cFileName);
```

Comment:
Do you want to keep these commented printf's ?

Reply:
Yes. They serve as 'debugging breadcrumbs' and make it easy to test that the code does what is expected.
But, @HBelusca, you can just say 'Let's remove them.' and I will be happy to do so. Thanks.

Reply:
It's ok if they don't become too many